README.md

```diff
@@ -42,13 +42,25 @@ load(
 
 apple_rules_dependencies()
 
+git_repository(
+    name = "build_bazel_rules_swift",
+    remote = "https://github.com/bazelbuild/rules_swift.git",
+    tag = "0.5.0",
+)
```

Comment:
these are already added in the `apple_rules_dependencies` method, please remove

Reply:
Do you say there's actually no need to load and call both `swift_rules_dependencies` and `apple_support_dependencies`, because those are handled with `apple_rules_dependencies` above?

Reply:
no, you need to call `swift_rules_dependencies` and `apple_rules_dependencies`, so those are ok... only the new `git_repository` should be removed

Reply:
Interesting... I remember there was an error saying a git repo such and such not found, so I had to load them explicitly. Now that's not the case, pulled.